_includes/promos/10tvpromo.html

```diff
@@ -90,14 +96,36 @@ <h4><br/>Offer valid until May 1st, 2020<br />To submit your business, fill out
                 if (forms.length > 0) {
                     for (var form of forms) {
                         form.addEventListener('submit', function(e) {
+                            console.log("Submitting");
                             if (!form.checkValidity()) {
                                 e.stopPropagation();
-                                
+                                e.preventDefault();
+                                console.log("FORM NOT VALID");
                             } else {
-                            submitText.style.color = 'black';
-                            submitButton.className = 'btn btn-success';
-                            submitButton.innerHTML = '&check;';
-                            submitButton.disabled = true;
+                                e.preventDefault();
+                                var xhr = new XMLHttpRequest();
+                                var myAPIKey = process.env.AIRTABLE_KEY;
```

Comment:
Where is the environment variable set? This code will execute on the client who likely doesn’t know our Airtable key.